snapcraft.yaml

```diff
@@ -12,14 +12,35 @@ description: |
 
 confinement: strict
 grade: stable
-base: core20
+base: core22
 
+plugs:
+  spread-config:
```

Comment:
This should be renamed `dot-spread` to match the conventions in https://snapcraft.io/docs/personal-files-interface